modules/data-entry/src/main/java/ca/sickkids/ccm/lfs/QueryBuilder.java

```diff
@@ -172,12 +172,15 @@ private String fullTextEscape(String input)
      */
     private String getQuestion(Resource res) throws ItemNotFoundException, RepositoryException
```

Comment:
```suggestion
    private String getQuestion(Resource res) throws ItemNotFoundException
```
If we're catching the `RepositoryException` in this function, it's no longer necessary to re-throw it.

Reply:
Done - see commit `5eb1a09` :heavy_check_mark: